asserts/account_key.go

```diff
@@ -21,9 +21,12 @@ package asserts
 
 import (
 	"fmt"
+	"regexp"
 	"time"
 )
 
+var ValidAccountKeyName = regexp.MustCompile(`^[-a-z0-9]+$`)
```

Comment:
(Incidentally: wow, that regexp really gets mangled by being interpreted as Markdown.)

The regexp was a direct quote from @niemeyer, and I don't know that we actually gain a whole lot by being picky about multiple consecutive dashes.  But I agree that not starting with a dash is probably a good idea in order to avoid command-line confusion with options, so I've refined it to `^[a-z0-9][-a-z0-9]*$`.